GiniSwitchSDK/Example/Tests/ExtractionTests.swift

```diff
@@ -11,46 +11,32 @@ import XCTest
 
 class ExtractionTests: XCTestCase {
```

Comment:
I will set this class as _final_, although it is a test class. (You may save some miliseconds in your Jenkins build)